docs/resources/becomingaverifier.md

```diff
@@ -0,0 +1,135 @@
+﻿---
+title: Guide to becoming a verifier
+sidebar_position: 5
+---
+
+<!-- How to ensure this appears under 'Technology'? -->
+
+Becoming a verifier is an integral aspect of digital trust ecosystems, enabling services to leverage digital credentials issued by trusted entities and held by individuals or organizations. By implementing verification capabilities, services can enhance security, streamline processes and improve user experiences while maintaining privacy and control for credential holders. The following steps provide a guide to becoming a verifier, but remember that this is not a one-size-fits-all approach. You may need to adapt these steps to your specific needs and the process is often iterative rather than strictly linear.
+
+## 1. Identify your verification use case
+
+This involves analyzing your current processes, understanding your specific needs and identifying where digital credential verification can add the most value.
+
+**Things to consider:**
+-   Review existing processes that could benefit from digital credential verification
+-   Identify pain points in current identity or attribute verification methods
+-   Take a step back and consider your ecosystem, understanding your needs and how digital credentials fit within it
+-   Evaluate which existing credentials would be most valuable to verify within your use case
+-   Consider whether your use case requires existing credentials or if you need to work with a potential issuer to create new ones
+    
+**Resources:**
+-   [Ideation Workshop](https://ditp-bc.atlassian.net/wiki/spaces/DTT/pages/206831634): A structured approach to help you brainstorm and refine your verification use case.
+-   [Examples of digital credentials](https://ditp-bc.atlassian.net/wiki/spaces/DTT/pages/209879044): Real-world scenarios demonstrating successful implementation of digital credential verification.
+-   [Ecosystem builder:](https://ditp-bc.atlassian.net/wiki/spaces/DTT/pages/20545537) Build an ecosystem map to visualize and understand how key actors, credentials and experiences in your use case fit within a digital trust ecosystem.
```

Comment:
make sure the ":" is not part of the linked text throughout all pages. ex. 
[Ecosystem Builder](https://ditp-bc.atlassian.net/wiki/spaces/DTT/pages/20545537): Build an ecosystem map to visualize and understand how key actors, credentials and experiences in your use case fit within a digital trust ecosystem.